twilight-model/src/http/channel_position.rs

```diff
@@ -0,0 +1,37 @@
+//! Models used when setting the channel positions over HTTP.
+
+use serde::{Deserialize, Serialize};
+
+use crate::id::{marker::ChannelMarker, Id};
+
+/// Used to update the position of channels over HTTP.
+///
+/// ## Note:
+/// The fields with `Option<Option<T>>` Will be `null` if they have
```

Comment:
```suggestion
/// The fields with `Option<Option<T>>` will be `null` if they have
```
and perhaps a newline after the heading